Dockerfile

```diff
@@ -29,6 +29,17 @@ ONBUILD ARG PGUSER=odoo
 ONBUILD ARG PGPASSWORD=odoopassword
 ONBUILD ARG PGHOST=db
 ONBUILD ARG PGDATABASE=prod
+ONBUILD ARG ODOO_MAX_CRON_THREADS=2
+ONBUILD ARG ODOO_WORKERS=0
```

Comment:
Since the image should be _production first_, I guess we can put 4 here, or at least 2 (minimum required for odoo to work and print reports).